make test for some classes in the ui and the useCase

Al-Qassim opened the pull request:
- MainMenuViewTest
- WelcomeViewTest
- LoginViewTest
- LoginUseCaseTest
- LogoutUseCaseTest
- GetLoggedInUserUseCaseTest

Al-Qassim committed:
make test for
- MainMenuViewTest
- WelcomeViewTest
- LoginViewTest
- LoginUseCaseTest
- LogoutUseCaseTest
- GetLoggedInUserUseCaseTest

ZainabAlnajjar commented on src/test/kotlin/ui/loginView/LoginViewTest.kt:
```diff
@@ -1,81 +1,99 @@
+
+    @Test
+    fun `should go to welcome when login fail`() {
+//        every { loginUseCase(any(), any()) } throws UserNotFoundException()
```
delete died code

ZainabAlnajjar reviewed (approved):
I think it would be better if we don't commit & push the commented testes, just don't select that file when you commit, so if someone else is working on it they won't get conflict, also it looks weird to see it modified in not related branch

Al-Qassim committed:
remove comment from test

Al-Qassim commented:
I comment that file because it was causing en error when running tests, and I don't want this problem to be pushed to the dev

the dev should run the tests with no problems

if someone else is working on it, he can uncomment it in his branch, fix it, then merge it to the dev

ZainabAlnajjar committed:
Merge remote-tracking branch 'origin/dev' into test/tests-for-welcome-login-and-mainMenu-view-and-use-cases

# Conflicts:
#	src/test/kotlin/logic/useCases/LoginUseCaseTest.kt

Al-Qassim committed:
Merge remote-tracking branch 'origin/dev' into test/tests-for-welcome-login-and-mainMenu-view-and-use-cases

# Conflicts:
#	src/main/kotlin/ui/loginView/LoginView.kt
#	src/test/kotlin/logic/useCases/LoginUseCaseTest.kt

Al-Qassim committed:
fix the tests for
- GetLoggedInUserUseCaseTest
- LoginViewTest
- LoginUseCaseTest
- MainMenuViewTest
- WelcomeViewTest
- LogoutUseCaseTest
